pkg/controllers/openshift-crd-manager.go

```diff
@@ -39,14 +39,14 @@ func (s *OpenShiftCRDManager) Dependencies() []string { return []string{"kube-ap
 func (s *OpenShiftCRDManager) Run(ctx context.Context, ready chan<- struct{}, stopped chan<- struct{}) error {
 	defer close(stopped)
 
-	if err := assets.ApplyCRDs(s.cfg); err != nil {
+	if err := assets.ApplyCRDs(s.cfg, ctx); err != nil {
```

Comment:
Can we change this so `ctx` is the first parameter, to keep in line with the other additions?